update_db.js

```diff
@@ -18,7 +21,7 @@ get('https://discord.com/api/v9/applications/detectable', res => {
   file.on('finish', () => {
     file.close();
 
-    const updated = JSON.parse(readFileSync(path, 'utf8'));
+    const updated = require(path);
```

Comment:
`require` caches so this will be the same as `current`, just leave this file untouched as it shouldn't be bundled anyway. Thanks!

Reply:
@CanadaHonk Oh wow, I wasn't aware that `require()` stores everything in a cache, thanks for catching that! I see the same issue now when trying to update and can confirm that it's not reloading the file. I'll fix it now by reverting that file. You're right that there's no point trying to load/use this particular file inside a bundle.